.github/pull_request_template.md

```diff
@@ -1,23 +1,23 @@
-**Thank you for submitting this pull request**
+**JIRA**: [link](https://www.example.com) 
 
-**JIRA**: _(please edit the JIRA link if it exists)_ 
+**Referenced Pull Requests** (if any):
 
-[link](https://www.example.com)
+**Checklist**:
 
-**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_
+- [ ] Documentation updated if applicable
+- [ ] Upgrade recipe provided if applicable
 
-paste the link(s) from GitHub here
+**Thank you for submitting this pull request!**
```

Comment:
Personally, I think that this is a good thing to start with.

Reply:
In my opinion we can keep it but comment it out. The person opening a PR will see it but it will remain invisible after the PR is submitted, reducing noise.